metricbeat/module/system/_meta/config.full.yml

```diff
@@ -16,7 +16,7 @@
     - filesystem
 
     # File system summary stats
-    #- fsstat
+    - fsstat
```

Comment:
It was working on Windows previously. So maybe it was commented out to reduce the amount of data Metricbeat sends? But the fsstats data is used on the Metricbeat overview dashboard which is why I think we should enabled it by default.